app/components/Button/Redesign/Button.jsx

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import React, { Component } from 'react'
+import { omit } from 'lodash'
+
+import styles from './Button.scss'
+import Login from '../../../images/icons/Login.svg'
+import Plus from '../../../images/icons/Plus.svg'
+import Wallet from '../../../images/icons/Wallet.svg'
+
+const iconMap = name => {
+  if (!name) {
+    return console.warn('iconMap in Button.jsx invoked without a valid icon!')
+  }
+
+  const icons = {
+    login: Login,
+    plus: Plus,
+    wallet: Wallet
+  }
+  return icons[name]
+}
+
+type Props = {
+  icon: ?string,
+  text: string,
+  dark: ?boolean,
+  light: ?boolean
+}
+
+class Button extends Component<Props> {
+  render = () => {
+    const { dark, light, text, icon } = this.props
+    const passDownProps = omit(this.props, 'dark', 'light', 'icon')
```

Comment:
So this might be semantically confusing but this is to actually render a dark or light button colored version of the new button. I can possibly change these to `primary` and `secondary`to be more explicit. Let me know what you think

Reply:
That sounds great if you're able to!